src/components/subscriptions/SubscriptionCard.jsx

```diff
@@ -26,17 +26,17 @@ const SubscriptionCard = ({
   return (
     <Card className="subscription-card w-100">
       <Card.Body>
-        <Card.Title>
+        <Card.Title className="mb-0">
           {title}
-          {isExpired && (
-            <div className="ml-2">
-              <Badge variant="danger">
-                Expired
-              </Badge>
-            </div>
-          )}
         </Card.Title>
-        <p className="small">
+        {isExpired && (
```

Comment:
These changes prevent the `Badge` from being on the same line, and wrapping with awkward left padding when the subscription plan name is longer.